### PR TITLE
releasefix/SUPP-853 Non Prod banner display

### PR DESCRIPTION
**Pull request by kandaj**
Fix to display non production banner for all environments  expect production

### CODE REVIEW

**Review by cdjreekie — APPROVED** [review]

**Review by cdjreekie — APPROVED** [review]
lgtm

File: src/pages/commonComponents/searchBar/SearchBar.js [hunk 1]
```diff
@@ -311,22 +311,13 @@ class SearchBar extends React.Component {
...
-			if (urlEnv === 'uatbeta') {
-				currentEnv = 'UAT BETA';
-			} else if (urlEnv === 'uat') {
-				currentEnv = 'UAT';
-			} else if (urlEnv === 'latest') {
-				currentEnv = 'LATEST';
-			}
-		}
+
+		const showUatBanner = process.env.REACT_APP_SHOW_NON_PROD_BANNER ? !!JSON.parse(process.env.REACT_APP_SHOW_NON_PROD_BANNER) : false;
```
Comment (ThinkingMonkeyLtd) [inline src/pages/commonComponents/searchBar/SearchBar.js:315]: Isn't this the same as const showUatBanner = process.env.REACT_APP_SHOW_NON_PROD_BANNER?

**Review by ThinkingMonkeyLtd — CHANGES_REQUESTED** [review]

**Review by ThinkingMonkeyLtd — APPROVED** [review]

**Review by dannitahdruk — APPROVED** [review]
lgtm!

File: src/pages/commonComponents/searchBar/SearchBar.js [hunk 2]
```diff
@@ -311,22 +311,13 @@ class SearchBar extends React.Component {
...
-			if (urlEnv === 'uatbeta') {
-				currentEnv = 'UAT BETA';
-			} else if (urlEnv === 'uat') {
-				currentEnv = 'UAT';
-			} else if (urlEnv === 'latest') {
-				currentEnv = 'LATEST';
-			}
-		}
+
+		const showUatBanner = urlEnv !== 'prod' ? true : false;
```
Comment (ThinkingMonkeyLtd) [inline src/pages/commonComponents/searchBar/SearchBar.js:315]: const showUatBanner = urlEnv !== 'prod' ? true : false; can just be const showUatBanner = urlEnv !== 'prod';

**Pull request closed without merging**